Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -230,19 +236,10 @@ void USpatialReceiver::CreateActor(Worker_EntityId EntityId)
 		}
 		else
 		{
-			// Either spawn the actor or get it from the level if it has a persistent name.
-			if (UnrealMetadataComponent->StaticPath.IsEmpty())
-			{
-				UE_LOG(LogTemp, Log, TEXT("!!! Spawning a native dynamic %s whilst checking out an entity."), *ActorClass->GetFullName());
-				EntityActor = SpawnNewEntity(PositionComponent, ActorClass, true);
-				bDoingDeferredSpawn = true;
-			}
-			else
-			{
-				FString FullPath = UnrealMetadataComponent->StaticPath;
-				UE_LOG(LogTemp, Log, TEXT("!!! Searching for a native static actor %s of class %s in the persistent level whilst checking out an entity."), *FullPath, *ActorClass->GetName());
-				EntityActor = FindObject<AActor>(World, *FullPath);
-			}
+			UE_LOG(LogTemp, Log, TEXT("!!! Spawning a native dynamic %s whilst checking out an entity."), *ActorClass->GetFullName());
```

Comment:
Drive-by: Can you make this not LogTemp?

Reply:
I'm planning a different PR that changes all of these `LogTemp`s to have proper log categories.